lib/fog/vcloud_director/models/compute/disks.rb

```diff
@@ -32,6 +32,11 @@ def item_list
           end
           items
         end
+        
+        # Filters out all the controllers (returns only resource_type == 17)
```

Comment:
Thanks for the suggestion. I've added a comment saying it's the OVF resource type and a link to a VMware blog on the subject.